meteor/client/ui/RundownView/RundownTiming/RundownTiming.ts

```diff
@@ -22,8 +23,10 @@ export namespace RundownTiming {
 	 * @enum {number}
 	 */
 	export enum Events {
+		/** Event is emitted once a second, to update displays in a synced manner */
+		'timeupdateSynced' = 'sofie:rundownTimeUpdateSynced',
 		/** Event is emitted every now-and-then, generally to be used for simple displays */
-		'timeupdate' = 'sofie:rundownTimeUpdate',
+		'timeupdateLR' = 'sofie:rundownTimeUpdateLR',
```

Comment:
I would prefer LowResolution instead of just LR and HighResolution instead of HR